packages/perseus/src/util/extract-perseus-data.ts

```diff
@@ -233,6 +232,28 @@ function getAnswersFromWidgets(
     return answers;
 }
 
+/**
+ * Join the content of the options together.
+ * @param {Array<{content: string}>} options
+ * @returns {string}
+ */
+const joinOptionContents = (options: readonly {content: string}[]): string =>
+    options.map(({content}) => content).join("\n");
+
+/**
+ * Convert an index to an option letter.
+ * @param {number} index
+ * @returns {string}
+ * @example
+ * toOptionLetter(0) // 'A'
+ * toOptionLetter(1) // 'B'
+ * toOptionLetter(25) // 'Z'
+ * // Once the index goes past 25, it will start returning special characters
+ * toOptionLetter(26) // '['
+ */
+const toOptionLetter = (index: number): string =>
```

Comment:
I'm late to this, but the function that the radio widget uses is already available for use: https://github.com/Khan/perseus/blob/53fd3768652b702df4a79b97a0ab4766b11748fe/packages/perseus/src/widgets/radio/util.ts#L8

Reply:
I see this one translates the letter. Since the rest of the prompt is not translated, I'd be worried about the inconsistency causing some confusion in the completion.

Reply:
i18n, that's a good point! I didn't notice this one wasn't translated. This differs in that it can extend past `Z`, but I am 99.9% sure that a radio widget with > 26 choices is problematic in many other ways! :)